modules/machine-pools-hcp.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="machine_pools_hcp_{context}"]
+= Machine pools in Hosted Control Plane (HCP) clusters
+
+In {product-title} (ROSA) HCP clusters, including Multi-AZ HCP clusters, machine pools only exist inside of a single Availability Zone. In Multi-AZ HCP clusters, ROSA creates a group of three machine pools in each Availability Zone. Additionally, machine pool version upgrades in HCP clusters do not track control plane upgrades. 
```

Comment:
Suggest to paraphrase as "Additionally, HCP machine pools can be upgraded independently from each other as well as from cluster's control plane.